lua/neo-tree/setup/init.lua

```diff
@@ -728,6 +728,8 @@ M.merge_config = function(user_config, is_auto_config)
 
   local rt = utils.get_value(M.config, "resize_timer_interval", 50, true)
   require("neo-tree.ui.renderer").resize_timer_interval = rt
+  --TODO how to initialize properly
+  require("neo-tree.sources.common.hijack_cursor").setup(M.config, nil)
```

Comment:
I wasn't able to figure out how this should be initialized. I would be grateful for some guidance here.

Reply:
Generally all initialization that is needed happens in setup.merge_config. There's no particular pattern to that except for the sources themselves that have to conform to an api.